R/create_config.R

```diff
@@ -60,7 +70,8 @@ create_config <- function(...) {
     ct_distribution = "norm",
     ct_distribution_params = c(mean = 25, sd = 2),
     network = "adjusted",
-    time_varying_death_risk = NULL
+    time_varying_death_risk = NULL,
+    reporting_delay = NULL
```

Comment:
I understand that `create_config()` is for modifying/specifying defaults that users won't often have to change, but I don't believe that's the case for `reporting_delay`. I think there is a case for it to be a "main" argument instead of being placed here. Doing so will better expose the functionality to the users for their control.

Reply:
I did deliberate on this when first implementing the feature and was on the fence. I agree with your point that it would be better to expose the functionality to the users, I've resolved in 44730ee52edd42a4a183172e6757f28ae3025d93.